extensions/google-shopping-ads/index.md

```diff
@@ -7,7 +7,7 @@ The Google Shopping ads extension installs and adds Google Shopping features to
 
 ## Requirements
 
-- **Magento Instance**: Google Shopping Ads Channel can be installed on instances with {{site.data.var.ce}} and {{site.data.var.ee}} versions 2.2.X and 2.3.X. We do not support the extension on Magento 2.1 or Magento 1.
+- **Magento Instance**: Google Shopping ads Channel can be installed on instances with {{site.data.var.ce}} and {{site.data.var.ee}} versions 2.2.X and 2.3.X. We do not support the extension on Magento 2.1 or Magento 1.
```

Comment:
Is this available for both Open Source and Commerce? If so, use similar language as line 24. If not, be specific and use a jekyll variable for the product name.